zng/union.go

```diff
@@ -44,25 +43,53 @@ func (t *TypeUnion) String() string {
 }
 
 func (t *TypeUnion) Parse(in []byte) (zcode.Bytes, error) {
+	panic("TypeUnion.Parse shouldn't be called")
+}
+
+// SplitBzng takes a bzng encoding of a value of the receiver's type and
+// returns the concrete type of the value, its index into the union
+// type, and the concrete value encoding.
```

Comment:
```suggestion
// type, and the raw value encoding.
```

Reply:
i just dropped "concrete" but didn't add raw as it seems to describe something slightly different...